lib/srv/ingress/reporter.go

```diff
@@ -0,0 +1,212 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+
+	"github.com/gravitational/trace"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/gravitational/teleport/lib/observability/metrics"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+// Constants for each ingress service.
+const (
+	Web         = "web"
+	SSH         = "ssh"
+	Kube        = "kube"
+	Tunnel      = "tunnel"
+	MySQL       = "mysql"
+	Postgres    = "postgres"
+	SQLServer   = "sqlserver"
```

Comment:
This is not used in the PR. Was recording metrics for this forgotten or is it to be done later?